CIScripts/Tests/VRouterAgentTests.ps1

```diff
@@ -29,28 +31,28 @@ function Test-VRouterAgentIntegration {
 
         Invoke-Command -Session $Session -ScriptBlock {
             $ControllerIP = $Using:ControllerIP
-            $VHostIfName = $Using:HNSTransparentIfName
-            $PhysIfName = $Using:PhysicalAdapterIfName
+            $VHostIfName = $Using:VHostIfName
+            $PhysIfName = $Using:PhysIfName
 
             $SourceConfigFilePath = $Using:SourceConfigFilePath
             $DestConfigFilePath = $Using:DestConfigFilePath
 
-            $ConfigFileContent = Get-Content $SourceConfigFilePath
+            $ConfigFileContent = [System.IO.File]::ReadAllText($SourceConfigFilePath)
```

Comment:
BTW why `Get-Content` doesn't work here?

Reply:
`Get-Content` does not handle Unix newlines correctly.